doc/reference/cmdline.rst

```diff
@@ -23,13 +38,13 @@ Running the server
 
 .. option:: -i <modules>, --init <modules>
 
-    comma-separated list of modules to install before running the server
-    (requires :option:`-d`).
+    comma-separated list of modules to install before running the server.
+    Use ``all`` for all modules. (requires :option:`-d`).
```

Comment:
Not actually a thing (you can try, `-i all` will just install base and whatever auto-install modules only depend on base).

Reply:
the description was taken from odoo/odoo/tools/config.py
```
        group.add_option("-i", "--init", dest="init", help="install one or more modules (comma-separated list, use \"all\" for all modules), requires -d")

```
Please advice how to adapt this. I will also include change config.py then to have consistency in both places.

Reply:
Not sure if the `all == base` behaviour is expected, let's not change this at this time and we can investigate it later (but maybe we should just remove the `all` option)